src/MigrationTools.Clients.AzureDevops.ObjectModel/Enrichers/TfsEmbededImagesEnricher.cs

```diff
@@ -48,71 +51,99 @@ public override int Enrich(WorkItemData sourceWorkItem, WorkItemData targetWorkI
 
         protected override void FixEmbededImages(WorkItemData wi, string oldTfsurl, string newTfsurl, string sourcePersonalAccessToken = "")
         {
-            Log.LogInformation("EmbededImagesRepairEnricher: Fixing HTML field attachments for work item {Id} from {OldTfsurl} to {NewTfsUrl}", wi.Id, oldTfsurl, GetUrlWithOppositeSchema(oldTfsurl));
+            var logTypeName = nameof(TfsEmbededImagesEnricher);
+            Log.LogInformation($"{logTypeName}: Fixing HTML field attachments for work item {wi.Id} from {oldTfsurl} to {newTfsurl}");
```

Comment:
Please return all "LogInformation" lines to the format `"test {item} and {item2}", item, itesm2`

These are used as parameters for telemetry and log hilights.

Reply:
Updated as requested..